src/sentiment_exploration.py

```diff
@@ -0,0 +1,27 @@
+import pandas as pd
+from textblob import TextBlob
+
+def detect_sentiment(text):
+    return TextBlob(text).sentiment.polarity
+
+def apply_sentiment_analysis(df):
+    # Apply the function to each element of the column
+    df.summary_cleaned.fillna('', inplace=True)
+    df['combined_text_Summary'] = df['text_cleaned'] + ' ' + df['summary_cleaned']
+    df['comb_sentiment'] = df.combined_text_Summary.apply(detect_sentiment)
+    return df
+
+def main():
+    # Load your DataFrame here
+    df = pd.read_csv("../data/Reviews_processed_10000.csv")
+
```

Comment:
try to use the DATA_DIR from settings.py

you can import it like this:
```
import sys
sys.path.append('.')

from settings import DATA_DIR
```

And then combine it with the rest of the path